src/polynomial/mod.rs

```diff
@@ -256,6 +256,59 @@ impl<F: FiniteField, const D: usize> Polynomial<Monomial, F, D> {
       }),
     )
   }
+
+  /// Computes the Fast Fourier Transform (FFT) of a polynomial in the Monomial basis.
```

Comment:
Can you make this and the other doc comment like the others in this module? It is lacking.